packages/shared/lib/core/network/types/network-metadata-map.type.ts

```diff
@@ -0,0 +1,6 @@
+import { NetworkId } from '../enums'
+import { IStardustNetworkMetadata } from '../interfaces'
+
+export type NetworkMetadataMap = {
```

Comment:
Do we really need a type for this in an external file?

Reply:
Yes, b/c it's used in multiple places. Define it once and you're done

Reply:
What are the multiple places? Its used in the map file and tests. Arguably we are doing the tests wrong duplicating this map or we could also just hard code the expected values for each test.